### PR TITLE
Added logrotate config.

### PR DESCRIPTION
**Pull request by juliusmilan**
Added simple logrotate config file with just default values, similar to one in retrace-server.

### CODE REVIEW

**Comment by trapas** [comment]
Can one of the admins verify this patch?

**Comment by xsuchy** [comment]
add to whitelist

File: config/faf [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/var/log/faf/* {
+    missingok
+    notifempty
+    weekly
+}
```
Comment (xsuchy) [inline config/faf]: Can you add `rotate 5`? So old logs are actually removed.

**Comment by marusak** [comment]
add to whitelist

**Comment by marusak** [comment]
#520

File: config/faf (hunk 1 above)
Comment (juliusmilan) [inline config/faf]: Done

**Comment by marusak** [comment]
close #520

**Pull request merged**